Jellyfin.Api/Controllers/StartupController.cs

```diff
@@ -30,22 +31,28 @@ public StartupController(IServerConfigurationManager config, IUserManager userMa
         }
 
         /// <summary>
-        /// Api endpoint for completing the startup wizard.
+        /// Completes the startup wizard.
         /// </summary>
+        /// <response code="200">Startup wizard completed.</response>
+        /// <returns>Status.</returns>
         [HttpPost("Complete")]
-        public void CompleteWizard()
+        [ProducesResponseType(StatusCodes.Status200OK)]
+        public ActionResult CompleteWizard()
         {
             _config.Configuration.IsStartupWizardCompleted = true;
             _config.SetOptimalValues();
             _config.SaveConfiguration();
+            return Ok();
         }
 
         /// <summary>
-        /// Endpoint for getting the initial startup wizard configuration.
+        /// Gets the initial startup wizard configuration.
         /// </summary>
+        /// <response code="200">Initial startup wizard configuration retrieved.</response>
         /// <returns>The initial startup wizard configuration.</returns>
```

Comment:
I think this might better match our convention with other routes. The object returned is also implicitly cast to an `ActionResult` so I think this is more accurate as well. Thoughts?

```suggestion
        /// <returns>An <see cref="OkResult"/> containing the initial startup wizard configuration.</returns>
```